client/core/core_test.go

```diff
@@ -2402,12 +2402,18 @@ func TestHandleRevokeOrderMsg(t *testing.T) {
 		rig.db, rig.queue, walletSet, tDcrWallet.fundingCoins, rig.core.notify)
 	rig.dc.trades[oid] = tracker
 
+	ch := orderNoteFeed(tCore, t)
+
 	// Success
 	err = handleRevokeOrderMsg(rig.core, rig.dc, req)
 	if err != nil {
 		t.Fatalf("handleRevokeOrderMsg error: %v", err)
 	}
 
+	details := fmt.Sprintf("Order %s on market %s at %s has been revoked by the server", tracker.token(), tracker.mktID, dc.acct.host)
+	expectedOrderNote := newOrderNote("Order Revoked", details, db.ErrorLevel, tracker.coreOrderInternal())
+	verifyRevokeNotification(ch, expectedOrderNote, t)
```

Comment:
You don't need to construct or compare an entire notification here. Subject comparison is enough.

Reply:
Passing subject only now.